packages/account/src/Components/currency-selector/currency-selector.tsx

```diff
@@ -1,294 +1,310 @@
 import classNames from 'classnames';
 import React, { HTMLAttributes, RefObject } from 'react';
```

Comment:
I'm not sure if I had asked you about it here, but why do we use types separately imported?